.env.example

```
@@ -68,15 +68,18 @@ S3_PUBLIC_URL=http://localhost:9000/twotter-media
 # ---------- Redis (rate-limit + realtime + cache, added at M2) ----------
 REDIS_URL=redis://localhost:6379
 
+# ---------- Databuddy analytics ----------
+# Client ID (from https://app.databuddy.cc). Leave blank to disable client-side analytics.
+DATABUDDY_CLIENT_ID=
+# Server-side API key (format: dbdy_xxx). Leave blank to disable server-side event tracking.
+DATABUDDY_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABUDDY_API_KEY key should go before the DATABUDDY_CLIENT_ID key

(UnorderedKey)